containers/build_and_push.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
+CURRENT_GIT_HASH=$(git rev-parse HEAD)
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+
+pushd $PROJECT_ROOT
+podman build -f ./containers/Containerfile -t quay.io/candlepin/subscription-manager:$CURRENT_GIT_HASH
+podman push --creds $QUAY_CREDS quay.io/candlepin/subscription-manager:$CURRENT_GIT_HASH
```

Comment:
I can see timout of Jenkins job, then we this script tries to push container to `quay.io`

BTW: why do we try to push container of each commit to the quay? Is it really necessary?

Reply:
should `$QUAY_CREDS` better be quoted too?